packages/medusa/src/api/routes/store/carts/index.ts

```diff
@@ -154,6 +154,52 @@ export const defaultStoreCartRelations = [
 /**
  * @schema StoreCartsRes
  * type: object
+ * x-expanded-relations:
+ *   field: cart
+ *   relations:
+ *     - billing_address
+ *     - discounts
+ *     - discounts.rule
+ *     - gift_cards
+ *     - items
+ *     - items.adjustments
+ *     - items.variant
+ *     - payment
+ *     - payment_sessions
+ *     - region
+ *     - region.countries
+ *     - region.payment_providers
+ *     - shipping_address
+ *     - shipping_methods
+ *     - shipping_methods.shipping_option
+ *   implicit:
+ *     - items.tax_lines
+ *     - items.variant.product
+ *     - region.fulfillment_providers
+ *     - region.payment_providers
+ *     - region.tax_rates
+ *     - shipping_methods.shipping_option
+ *     - shipping_methods.tax_lines
+ *   totals:
```

Comment:
question: Can we make this into a schema? This will be repeated many times I imagine.

Reply:
There would be a way to optimize but it would be through the introduction of a new entity. Something like `type OrderWithTotals exends Order` and we would use that new entity throughout our codebase as well. 

But I'm not sure we need such an optimization yet. While we have many endpoints that returns models with calculated totals, most of their response shapes are consolidated into only a few schema responses. For example, `StoreCartsRes` in used in 14 endpoints.

Reply:
Yes, I agree. Let's keep it as is and consider the totals type at a later point in time :)